internal/upboundagent/proxy.go

```diff
@@ -28,7 +28,7 @@ import (
 	"syscall"
 	"time"
 
-	"github.com/dgrijalva/jwt-go"
+	"github.com/golang-jwt/jwt"
```

Comment:
We still refer to the old library (https://github.com/dgrijalva) in a comment at line 438) in this file in the context of a previous fix. Because it's about clock skew, so I would expect that issue to be also relevant with the new library but I have not put thought into it. Just writing here so that we do not overlook it.

Reply:
This should be ok. We already fixed that problem on our end and the issue is referenced just to set the context.